notebooks/community/gapic/custom/showcase_custom_image_classification_online_explain_example_based_api.ipynb

```diff
@@ -0,0 +1,3291 @@
+{
```

Comment:
<p>Vertex client library &gt; Vertex AI client library</p><p>Vertex server &gt; Vertex AI server</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/716/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>

Reply:
<p>Also, what is a Vertex AI server? We should refer to the specific service.</p>

Reply:
<p>I rephased for simplification. </p>

Reply:
<p>done</p>